src/framework/theme/components/icon/icon-libraries.ts

```diff
@@ -181,7 +181,7 @@ export class NbIconLibraries {
 
   protected getIconFromPack(name: string, pack: NbIconPack, shouldThrow = true): NbIcon | string {
     if (shouldThrow && !pack.icons.has(name)) {
-      throwIconNotFoundError(name, pack.name);
+      return null
     }
 
     return pack.icons.get(name);
```

Comment:
Please, also remove `shouldThrow` argument as it's part of old behaviour. And then you could simplify method to:
```
if (pack.icons.has(name)) {
  return pack.icons.get(name);
}

return null;
```
And add null as possible return type.